ServerProfile/Trader/TraderConfig.txt

		/////////////////////////////////////////////////////////////////////////////////////////////////
        //      Need Help? The Trader Mod has its own Channel on the DayZ Modders Discord Server!      //
        //                                                                                             //
        //      Only Singleline Comments work. Don't use Multiline Comments!                           //
        //      /* THIS COMMENT WILL CRASH THE SERVER! */                                              //
        //      // THIS COMMENT WORKS!                                                                 //
        /////////////////////////////////////////////////////////////////////////////////////////////////

<CurrencyName> #tm_ruble							// Currency Name (Only used for the Text in the upper right Corner of the Trader Menu)
	<Currency> MoneyRuble1, 	1					// Currency Classname, Currency Money Value
    <Currency> MoneyRuble5, 	5
    <Currency> MoneyRuble10, 	10
    <Currency> MoneyRuble25, 	25
    <Currency> MoneyRuble50, 	50
    <Currency> MoneyRuble100, 	100


<Trader> Consume Trader							// Trader Name (first Trader has ID = 0, second Trader has ID = 1, and so on..)
 	<Category> Vegetables							// Category Name
 		CaninaBerry,			*,		1,		-1	// Item Classname, Quantity, Buyvalue, Sellvalue
        SambucusBerry,			*,		1,		-1	// Quantity * means max value; Quantity V means Vehicle; Quantity VNK means Vehicle without Key; Quantity M means Magazine; Quantity W means Weapon; Quantity S means Steack Meat; Quantity K means Key Duplication
        Plum,					*,		1,		-1	// Buyvalue -1 means it can not be bought
        Kiwi,					*,		1,		-1	// Sellvalue -1 means it can not be selled
        Apple,					*,		1,		-1
        Pear,					*,		1,		-1
        Orange,					*,		1,		-1
        Banana,					*,		1,		-1
        Tomato,					*,		1,		-1
        Potato,					*,		1,		-1
        GreenBellPepper,		*,		1,		-1
        Zucchini,				*,		3,		1
        Pumpkin,				*,		8,		3
        AgaricusMushroom,		*,		2,		1
        AmanitaMushroom,		*,		2,		1
        LactariusMushroom,		*,		2,		1
        MacrolepiotaMushroom,	*,		2,		1
        PsilocybeMushroom,		*,		2,		1
        AuriculariaMushroom,	*,		2,		1
        BoletusMushroom,		*,		2,		1
        PleurotusMushroom,		*,		2,		1

    <Category> Seeds & Lime
        TomatoSeedsPack,		*,		4, 		1
        PepperSeedsPack,		*,		4, 		1
        ZucchiniSeedsPack,		*,		6, 		2
        PumpkinSeedsPack,		*,		10, 	3
        PotatoSeed,		        *,		10, 	3
        GardenLime,				*,		20,		7


    <Category> Packaged Food
		PowderedMilk,			*,		1,		-1
		BoxCerealCrunchin,		*,		2,		1
		Rice,					*,		3,		1
		Marmalade,				*,		3,		1
		Honey,				    *,		3,		1
		PeachesCan,				*,		4,		2
		SardinesCan,			*,		4,		2
		TunaCan,				*,		4,		2
		DogFoodCan,				*,		4,		2
		CatFoodCan,				*,		4,		2
		PorkCan,				*,		4,		2
		Lunchmeat,				*,		4,		2
		Pate,				    *,		4,		2
		Pajka,				    *,		4,		2
		SpaghettiCan,			*,		5,		2
		BakedBeansCan,			*,		5,		2
		TacticalBaconCan,		*,		5,		3
		SaltySticks,		    *,		5,		3
		BrisketSpread,		    *,		5,		3
		Crackers,		        *,		5,		3
		Chips,		            *,		5,		3
		ZagorkyPeanuts,		    *,		5,		3
		ZagorkyChocolate,		*,		5,		3
		Zagorky,		        *,		5,		3

	<Category> Meat
    		ChickenBreastMeat,		S,		2,		1
    		GoatSteakMeat,			S,		3,		1
    		SheepSteakMeat,			S,		3,		1
    		PigSteakMeat,			S,		4,		2
    		BoarSteakMeat,			S,		4,		2
    		WolfSteakMeat,			S,		4,		2
    		DeerSteakMeat,			S,		5,		2
    		CowSteakMeat,			S,		5,		2
    		Sardines,				S,		7,		4
    		Carp,					S,		10,		5
    		Mackerel,				S,		10,		5
		
	<Category> Animal Parts
    		Bone,					*,		2,		1
    		Lard,					*,		2,		1
    		Guts,					*,		2,		1
    		SmallGuts,				*,		2,		1

    	<Category> Drinks
    		SodaCan_Cola, 			*,		2, 		1
    		SodaCan_Fronta, 		*,		2, 		1
    		SodaCan_Cola, 			*,		2, 		1
    		SodaCan_Kvass, 			*,		2, 		1
    		SodaCan_Pipsi, 			*,		2, 		1
    		SodaCan_Spite, 			*,		2, 		1
    		WaterBottle,			*,		5,		2
    		Canteen,				*,		8,		4
    		Vodka,					*,		12,		8

    	<Category> Medical Supplies
    		//Rag,					1,		1,		-1
    		BandageDressing,		2,		3,		1
    		BloodTestKit,			*,		5,		2
    		//InjectionVial,		*,		5, 		2
    		Thermometer,			*,		5,		2
    		DisinfectantSpray,		*,		8,		5
    		//Syringe,				*,		8, 		5
    		Heatpack,				*,		10,		5
    		StartKitIV,				*,		10,		6
    		SalineBag,				*,		15,		8
    		BloodBagEmpty,			*,		18,		10
    		FirstAidKit,			*,		20,		10
    		//Defibrillator,		*,		35, 	20
    		AntiChemInjector,		*,		20,		10

    	<Category> Medications
    		CharcoalTablets,		*,		2,		1
    		DisinfectantAlcohol,	*,		4,		2
    		IodineTincture,	        *,		4,		2
    		VitaminBottle,			*,		6,		4
    		TetracyclineAntibiotics,*,		8,		5
    		PainkillerTablets,		*,		10,		6
    		Epinephrine,			*,		20,		10
    		Morphine,				*,		25,		15

    	<Category> Money Exchange
    		MoneyRuble1,			1,		1,		-1
    		MoneyRuble5,			1,		5,		-1
    		MoneyRuble10,			1,		10,		-1
    		MoneyRuble25,			1,		25,		-1
    		MoneyRuble50,			1,		50,		-1
    		MoneyRuble100,			1,		100,	-1


    <Trader> Misc Trader
    	<Category> Tools (small)
    		Screwdriver,			*,		3,		1
    		Wrench,					*,		3,		1
    		SewingKit,				*,		4, 		1
    		Pliers,					*,		4, 		1
    		LeatherSewingKit,		*,		5, 		2
    		Blowtorch,		        *,		5, 		2
    		Whetstone,				*,		5,		2
    		Hammer,					*,		7,		3
    		Hacksaw,				*,		7,		3
    		HandSaw,				*,		7,		3
    		Sickle,				    *,		7,		3
    		WeaponCleaningKit,		*,		8, 		3
    		KitchenKnife,			*,		8,		4
    		Cleaver,			    *,		8,		4
    		SteakKnife,				*,		9,		4
    		HuntingKnife,			*,		9,		4
    		CanOpener,				*,		10,		4
    		Compass,				*,		10,		4
    		OrienteeringCompass,	*,		10,		4
    		Hatchet,				*,		10,		4
    		CombatKnife,			*,		10,		5
    		Machete,				*,		12,		6
    		ChernarusMap,			*,		15,		5
    		Lockpick,				*,		20,		10
    		Binoculars,				*,		30,		15
    		FishingRod,				*,		60,		25
    		Hook,				    *,		60,		25
    		Bait,				    *,		60,		25

    	<Category> Tools (big)
    		LugWrench,				*,		4,		1
    		Crowbar,				*,		5,		3
    		Shovel,					*,		6,		3
    		FarmingHoe,				*,		6,		3
    		Pitchfork,				*,		6,		3
    		Pickaxe,				*,		10,		4
    		SledgeHammer,			*,		12,		6
    		PipeWrench,				W,		15,		7
    		WoodAxe,				*,		20,		8
    		FirefighterAxe,			*,		30,		12
    		Broom,				    *,		6,		3

    	<Category> Electronics
    		Battery9V,				*,		8,		3
    		PersonalRadio,			*,		10,		5
    		GPSReceiver,			*,		10,		5
    		Megaphone,				*,		30,		15
    		ElectronicRepairKit,	*,		35,		17
    		CableReel,				*,		40,		20
    		BatteryCharger,			*,		40,		20
    		BaseRadio,				*,		50,		30
    		Rangefinder,			*,		60,		35
    		NVGoggles,				*,		400,	220

    	<Category> Fire & Lights
    		Chemlight_Blue,			*,		2,		1
    		Chemlight_Green,		*,		2,		1
    		Chemlight_Red,			*,		2,		1
    		Chemlight_White,		*,		2,		1
    		Chemlight_Yellow,		*,		2,		1
    		Roadflare,				*,		4,		2
    		Matchbox,				*,		6,		3
    		PetrolLighter,			*,		6,		3
    		Flashlight,				*,		15,		5
    		PortableGasLamp,		*,		25,		10
    		PortableGasStove,		*,		30,		15
    		Headtorch_Black,		*,		40,		20
    		Headtorch_Grey,			*,		40,		20
    		Spotlight,				*,		600,	200
    		FireworksLauncher,		*,		600,	200

    	<Category> Cooking & Hunting Supplies
    		PurificationTablets,	*,		5,		2
    		Pot,					M,		25,		10
    		FryingPan,				M,		25,		10
    		Tripod,					*,		35,		15
    		BearTrap,				*,		50,		30

    	<Category> Hardware Supplies
    		Paper,					*,		1,		0
    		Pen_Blue,				*,		2,		1
    		Rope,					*,		4,		1
    		TannedLeather,			*,		4,		1
    		AntiPestsSpray,			*,		8,		4
    		MetalWire,				*,		15,		6
    		EpoxyPutty,				*,		15,		6
    		DuctTape,				*,		20,		8
    		XmasLights,				*,		25,		10
    		HandcuffKeys,			*,		35,		12
    		Handcuffs,				*,		50,		15
    		Netting,				1,		50,		10
    		BurlapSack,				*,		50,		10
    		WoodenPlank,			10,		50,		15
    		MetalPlate,				*,		60,		18
    		SmallProtectorCase,		*,		70,		30
    		AmmoBox,				*,		80,		35
    		AlarmClock_Blue,		*,		80,		35
    		AlarmClock_Green,		*,		80,		35
    		AlarmClock_Red,			*,		80,		35
    		KitchenTimer,			*,		80,		35
    		SmallGasCanister,		*,		80,		35
    		MediumGasCanister,		*,		100,	40
    		LargeGasCanister,		*,		200,	70
    		CanisterGasoline,		*,		300,	80
    		NailBox,				*,		300,	120
    		BarbedWire,				*,		450,	200
    		Camonet,				*,		600,	250
    		HescoBox,				*,		650,	250
    		CombinationLock,		*,		700,	300
    		CombinationLock4,		*,		900,	400
    		BarrelHoles_Blue,		*,		800,	300
    		BarrelHoles_Green,		*,		800,	300
    		BarrelHoles_Red,		*,		800,	300
    		BarrelHoles_Yellow,		*,		800,	300
    		Barrel_Blue,			M,		1500,	350
    		Barrel_Green,			M,		1500,	350
    		Barrel_Red,				M,		1500,	350
    		Barrel_Yellow,			M,		1500,	350
    		SeaChest,				*,		2000,	400
    		PowerGenerator,			*,		3000,	1000
    		MediumTent,				*,		3000,	500
    		MediumTent_Orange,		*,		3000,	500
            MediumTent_Green,		*,		3000,	500
    		PartyTent_Lunapark,		*,		3000,	500
    		PartyTent_Brown,		*,		3000,	500
            PartyTent_Blue,		    *,		3000,	500
    		PartyTent,		        *,		3000,	500
    		CarTent,				*,		5000,	700
    		LargeTent,				*,		7000,	1200

    	<Category> Fluffy Pets <3
    		Bear_Beige,				*,		5,		1
    		Bear_Dark,				*,		5,		1
    		Bear_Pink,				*,		5,		1
    		Bear_White,				*,		5,		1


    <Trader> Weapon Trader
    	<Category> Melee
    		BrassKnuckles_Shiny,	W,		15,		6
    		BrassKnuckles_Dull,	    W,		15,		6
    		StunBaton,				W,		15,		9
    		Pipe,					W,		25,		10
    		CattleProd,				W,		25,		15
    		BaseballBat,			W,		35,		17
    		MeatTenderizer,			W,		35,		17
    		NailedBaseballBat,		W,		45,		20
    		OrientalMachete,		W,		45,		20
    		KukriKnife,		        W,		45,		20
    		FangeKnife,		        W,		45,		20
    		CrudeMachete,		    W,		45,		20
    		BarbedBaseballBat,		W,		45,		20

    	<Category> Sidearms
    		Flaregun,               W,		80,		40
    		MakarovIJ70,			W,		100,	50
    		CZ75,					W,		135,	70
    		FNX45,					W,		175,	90
    		Glock19,				W,		300,	130
    		Derringer_Grey,			W,		300,	130
    		P1,				        W,		300,	130
    		MKII,					W,		320,	160
    		Colt1911,				W,		340,	180
    		Engraved1911,			W,		340,	180
    		Magnum,			        W,		340,	180
    		SawedoffMagnum,			W,		340,	180
    		Longhorn,			    W,		640,	180
    		Deagle,			        W,		640,	180
    		Deagle_Gold,			W,		640,	180

    	<Category> Rifles
    	    Ruger1022,				W,		350,	175
    		Izh18,					W,		350,	175
    		SawedoffIzh18,			W,		350,	175
    		SawedoffMosin9130,		W,		400,	200
    		Mosin9130,				W,		400,	200
    		CZ527,					W,		500,	250
    		Winchester70,			W,		500,	250
    		SKS,					W,		600,	300
    		Scout,          		W,		1600,	960
            Scout_Chernarus,		W,		1600,	960
            Scout_Livonia,		    W,		1600,	960
            Repeater,		        W,		1600,	960
            CZ550,		            W,		1600,	960

    	<Category> Shotguns
    		Mp133Shotgun,			W,		500,	250
    		SawedoffIzh43Shotgun,	W,		900,	400
    		Izh43Shotgun,			W,		900,	400
    		SawedoffIzh18Shotgun,	W,		900,	400
    		Izh18Shotgun,			W,		900,	400
    		Saiga,					W,		1500,	600

    	<Category> Submachine Guns
    		CZ61,					W,		700,	300
    		UMP45,					W,		800,	350
    		MP5K,					W,		1000,	450
    		PP19,					W,		1000,	450
    		AKS74U,					W,		1100,	600

    	<Category> Assault Rifles
    		FAL,					W,		1000,	450
    		AKM,					W,		1500,	750
    		AK101,					W,		1500,	750
    		AK74,					W,		1600,	750
    		M4A1,					W,		1500,	750
    		M16A2,					W,		1500,	750
    		FAMAS,			        W,		1500,	750
    		AugShort,			    W,		1500,	750
    		Aug,			        W,		1500,	750
    		SawedoffFAMAS,			W,		1500,	750

    	<Category> Sniper Rifles
    		VSS,					W,		1600,	960
    		B95,					W,		1700,	1100
    		SawedoffB95,			W,		1700,	1100
    		SSG82,			        W,		1700,	1100
    		SVD,					W,		2000,	1200
    		ASVAL,					W,		2000,	1200

    	<Category> Grenades & Launchers
    		RGD5Grenade,			1,		300,	100
    		M67Grenade,				1,		300,	100
    		M18SmokeGrenade_Red,	1,		300,	100
    		M18SmokeGrenade_Green,	1,		300,	100
    		M18SmokeGrenade_Yellow,	1,		300,	100
    		M18SmokeGrenade_Purple,	1,		300,	100
    		M18SmokeGrenade_White,	1,		300,	100
    		RDG2SmokeGrenade_Black,	1,		300,	100
    		RDG2SmokeGrenade_White,	1,		300,	100
    		FlashGrenade,			1,		300,	100
    		Ammo_40mm_Chemgas,		1,		300,	100
    		Ammo_40mm_Explosive,	1,		300,	100
    		Ammo_40mm_Smoke_Black,	1,		300,	100
    		Ammo_40mm_Smoke_Green,	1,		300,	100
    		Ammo_40mm_Smoke_Red,	1,		300,	100
    		Ammo_40mm_Smoke_White,	1,		300,	100
    		M79,					W,		2000,	1200


    <Trader> Weapon Supplies
    	<Category> Ammunition
    		Ammo_380,				1,		1,		-1
    		Ammo_9x19,				1,		1,		-1
    		Ammo_22,				1,		1,		-1
    		Ammo_45ACP,				1,		1,		-1
    		Ammo_12gaRubberSlug,	1,		3,		1
    		Ammo_12gaPellets,		1,		3,		1
    		Ammo_12gaSlug,			1,		4,		2
    		Ammo_9x39,				1,		4,		2
    		Ammo_9x39AP,			1,		5,		3
    		Ammo_308Win,			1,		5,		3
    		Ammo_357,			    1,		5,		3
    		Ammo_545x39,			1,		5,		3
    		Ammo_556x45,			1,		5,		3
    		Ammo_762x39,			1,		5,		3
    		Ammo_762x54,			1,		10,		6
    		Ammo_556x45Tracer,		1,		10,		5
    		Ammo_545x39Tracer,		1,		10,		5
    		Ammo_308WinTracer,		1,		10,		5
    		Ammo_762x39Tracer,		1,		10,		5
    		Ammo_Flare,				1,		15,		7
    	    Ammo_FlareBlue,			1,		15,		7
    	    Ammo_FlareGreen,		1,		15,		7
    	    Ammo_FlareRed,			1,		15,		7
    		Ammo_762x54Tracer,		1,		20,		12
    		// When changing Prices here keep in Mind that Players
    		// can just unpack the Ammoboxes and sell the Ammo seperatly!
    		// Originally the Ammoboxes buying Price is 20% less than
    		// buying single Ammo. The Selling Price is the same as
    		// selling the Ammo seperately.
    		AmmoBox_380_35rnd,		        *,		28,		-1
    		AmmoBox_9x19_25rnd,		        *,		20,		-1
    		AmmoBox_45ACP_25rnd,	        *,		20,		-1
    		AmmoBox_22_50Rnd,		        *,		60,		20
    		AmmoBox_12gaRubberSlug_10Rnd,	*,		24,		10
    		AmmoBox_00buck_10rnd,	        *,		24,		10
    		AmmoBox_12gaSlug_10Rnd,	        *,		32,		20
    		AmmoBox_9x39_20Rnd,		        *,		32,		20
    		AmmoBox_9x39AP_20Rnd,	        *,		80,		60
    		AmmoBox_308Win_20Rnd,	        *,		80,		60
    		AmmoBox_545x39_20Rnd,	        *,		80,		60
    		AmmoBox_556x45_20Rnd,	        *,		80,		60
    		AmmoBox_762x39_20Rnd,	        *,		80,		60
    		AmmoBox_762x54_20Rnd,	        *,		200,	120
    		AmmoBox_308WinTracer_20Rnd,		*,		160,	80
    		AmmoBox_357_20Rnd,		        *,		160,	80
    		AmmoBox_545x39Tracer_20Rnd,		*,		160, 	80
    		AmmoBox_556x45Tracer_20Rnd,		*,		160,	80
    		AmmoBox_762x39Tracer_20Rnd,		*,		160,	80
    		AmmoBox_762x54Tracer_20Rnd,		*,		400,	200

    	<Category> Magazines
    	    Mag_Ruger1022_15Rnd,	M,		20,		15
    	    Mag_Ruger1022_30Rnd,	M,		30,		15
    	    Mag_P1_8Rnd,			M,		30,		15
    		Mag_IJ70_8Rnd,			M,		30,		15
    		Mag_CZ75_15Rnd,			M,		40,		20
    		Mag_Glock_15Rnd,		M,		40,		20
    		Mag_MKII_10Rnd,			M,		40,		20
    		Mag_1911_7Rnd,			M,		40,		20
    		Mag_FNX45_15Rnd,		M,		40,		20
    		Mag_Deagle_9rnd,		M,		60,		20
    		Mag_CZ527_5rnd,			M,		60,		30
    		Mag_CZ550_10Rnd,		M,		100,	30
    		Mag_CZ61_20Rnd,			M,		100,	50
    		Mag_UMP_25Rnd,			M,		150,	75
    		Mag_MP5_15Rnd,			M,		100,	100
    		Mag_MP5_30Rnd,			M,		200,	100
    		Mag_PP19_64Rnd,			M,		200,	100
    		Mag_FAL_20Rnd,			M,		100,	50
    		Mag_Saiga_5Rnd,			M,		100,	50
    		Mag_Saiga_8Rnd,			M,		220,	60
    		Mag_Saiga_Drum20Rnd,	M,		480,	110
    		Mag_AKM_30Rnd,			M,		220,	110
    		Mag_AKM_Palm30Rnd,		M,		220,	110
    		Mag_AKM_Drum75Rnd,		M,		480,	220
    		Mag_AK101_30Rnd,		M,		220,	110
    		Mag_AK74_30Rnd,			M,		220,	110
    		Mag_AK74_45Rnd,			M,		220,	110
    		Mag_FAMAS_25Rnd,		M,		220,	110
    		Mag_Aug_30Rnd,		    M,		220,	110
    		Mag_STANAG_30Rnd,		M,		220,	110
    		Mag_STANAGCoupled_30Rnd,M,		480,	220
    		Mag_STANAG_60Rnd,       M,		880,	220
    		Mag_CMAG_10Rnd,			M,		120,	60
    		Mag_CMAG_20Rnd,			M,		180,	80
    		Mag_CMAG_30Rnd,			M,		220,	110
    		Mag_CMAG_40Rnd,			M,		300,	150
    		Mag_Scout_5Rnd,			M,		220,	110
    		Mag_SSG82_5rnd,			M,		220,	110
    		Mag_VSS_10Rnd,			M,		220,	110
    		Mag_VAL_20Rnd,			M,		220,	110
    		Mag_SVD_10Rnd,			M,		250,	125
    		Mag_308WinSnaploader_2Rnd,		M,		250,	125

    	<Category> Handguards
    		MP5_PlasticHndgrd,		*,		100,	50
    		MP5_RailHndgrd,			*,		150,	75
    		AK74_Hndgrd,			*,		150,	75
    		AK_WoodHndgrd,			*,		200,	100
    		AK_RailHndgrd,			*,		200,	100
    		AK_PlasticHndgrd,		*,		200,	100
    		M4_PlasticHndgrd,		*,		150,	75
    		M4_RISHndgrd,			*,		200,	100
    		M4_MPHndgrd,			*,		200,	100

    	<Category> Buttstocks
    		MP5k_StockBttstck,		*,		100,	50
    		PP19_Bttstck,		    *,		100,	50
    		Fal_OeBttstck,			*,		100,	50
    		Fal_FoldingBttstck,		*,		150,	75
    		Saiga_Bttstck,			*,		150,	75
    		AKS74U_Bttstck,			*,		150,	75
    		AK74_WoodBttstck,		*,		150,	75
    		AK_PlasticBttstck,		*,		100,	50
    		AK_WoodBttstck,			*,		150,	75
    		AK_FoldingBttstck,		*,		250,	125
    		M4_OEBttstck,			*,		150,	75
    		M4_CQBBttstck,			*,		150,	75
    		M4_MPBttstck,			*,		150,	75

    	<Category> Optics
    		M4_CarryHandleOptic,	*,		150,	60
    		BUISOptic,				*,		150,	60
    		PistolOptic,			*,		150,	60
    		FNP45_MRDSOptic,		*,		250,	100
    		ACOGOptic,				*,		300,	120
    		ACOGOptic,				*,		300,	120
    		ReflexOptic,			*,		300,	120
    		M68Optic,				*,		500,	200
    		M4_T3NRDSOptic,			*,		500,	200
    		KobraOptic,				*,		600,	250
    		KashtanOptic,			*,		700,	300
    		PUScopeOptic,			*,		900,	500
    		HuntingOptic,			*,		1000,	550
    		PSO1Optic,				*,		1100,	600
    		PSO11Optic,				*,		1400,	840
    		KazuarOptic,			*,		2200,	1400
    		StarlightOptic,			*,		2200,	1400

    	<Category> Supressors
    		PistolSuppressor,		*,		150,	75
    		AK_Suppressor,			*,		250,	125
    		M4_Suppressor,			*,		250,	125

    	<Category> Compensators
    		Mosin_Compensator,		*,		150,	75
    		MP5_Compensator,		*,		200,	100

    	<Category> Bayonets
    		Mosin_Bayonet,			*,		100,	60
    		M9A1_Bayonet,			*,		150,	75
    		AK_Bayonet,				*,		150,	75
    		SKS_Bayonet,			*,		100,	60

    	<Category> Flashlights
    		Universallight,			*,		120,	60
    		TLRLight,			    *,		120,	60

    <Trader> Clothing Trader
    	<Category> Glasses
    		SportGlasses_Orange,	*,		17,		8
    		SportGlasses_Blue,		*,		18,		9
    		SportGlasses_Black,		*,		20,		10
    		SportGlasses_Green,		*,		20,		10
    		ThinFramesGlasses,		*,		20,		10
    		ThickFramesGlasses,		*,		23,		11
    		DesignerGlasses,		*,		30,		15
    		AviatorGlasses,			*,		40,		20
    		EyePatch_Improvised,	*,		10,		8
    		TacticalGoggles,		*,		60,		30
    		NVGHeadstrap,			*,		200,	120

    	<Category> Armbands
    		Armband_Pink,			*,		15,		5
    		Armband_Yellow,			*,		15,		5
    		Armband_Orange,			*,		15,		5
    		Armband_Red,			*,		15,		5
    		Armband_Blue,			*,		15,		5
    		Armband_White,			*,		15,		5
    		Armband_Black,			*,		15,		5
    		Armband_Green,			*,		15,		5

    	<Category> Gloves
    		SurgicalGloves_LightBlue,*,		5,		1
    		SurgicalGloves_Blue,	*,		5,		1
    		SurgicalGloves_White,	*,		5,		1
    		SurgicalGloves_Green,	*,		5,		1
    		WorkingGloves_Yellow,	*,		25,		6
    		WorkingGloves_Beige,	*,		30,		8
    		WorkingGloves_Brown,	*,		30,		8
    		WorkingGloves_Black,	*,		30,		8
    		TacticalGloves_Beige,	*,		35,		10
    		TacticalGloves_Black,	*,		35,		10
    		TacticalGloves_Green,	*,		35,		10
    		OMNOGloves_Gray,		*,		40,		12
    		OMNOGloves_Brown,		*,		40,		12

    	<Category> Hats & Caps

Trader/config.cpp
////////////////////////////////////////////////////////////////////
//DeRap: config.bin
//Produced from mikero's Dos Tools Dll version 8.90
//https://mikero.bytex.digital/Downloads
//'now' is Thu Nov 10 04:05:45 2022 : 'file' last modified on Mon Feb 15 22:17:34 2021
////////////////////////////////////////////////////////////////////

#define _ARMA_

class CfgMods
{
	class Trader
	{
		dir = "Trader";
		picture = "";
		action = "";
		hideName = 1;
		hidePicture = 1;
		name = "";
		credits = "";
		author = "Dr_J0nes";
		authorID = "0";
		version = "1.0";
		extra = 0;
		type = "mod";
		dependencies[] = {"Game","World","Mission"};
		class defs
		{
			class gameScriptModule
			{
				value = "";
				files[] = {"TM/Trader/scripts/defines","TM/Trader/scripts/3_Game"};
			};
			class worldScriptModule
			{
				value = "";
				files[] = {"TM/Trader/scripts/defines","TM/Trader/scripts/4_World"};
			};
			class missionScriptModule
			{
				value = "";
				files[] = {"TM/Trader/scripts/defines","TM/Trader/scripts/5_Mission"};
			};
		};
	};
};
class CfgPatches
{
	class trader
	{
		units[] = {"MoneyRuble1","MoneyRuble5","MoneyRuble10","MoneyRuble25","MoneyRuble50","MoneyRuble100","Land_RoadCone","Hoodie_GraffitiTiles","Hoodie_DrJ0nes","VehicleKeyBase","VehicleKeyDuplicate","VehicleKeyLost","VehicleKeyRed","VehicleKeyBlack","VehicleKeyGrayCyan","VehicleKeyYellow","VehicleKeyPurple"};
		weapons[] = {};
		requiredVersion = 0.1;
		requiredAddons[] = {"DZ_Data"};
	};
};
class CfgModels
{
	class ruble
	{
		sections[] = {"WholeObject"};
	};
	class keyLada
	{
		sections[] = {"WholeObject"};
	};
};
class CfgVehicles
{
	class Inventory_Base;
	class MoneyRuble1: Inventory_Base
	{
		scope = 2;
		displayName = "$STR_tm_1_ruble_note";
		descriptionShort = "$STR_tm_ruble_description";
		model = "TM\Trader\ruble.p3d";
		canBeSplit = 1;
		rotationFlags = 16;
		lootCategory = "Materials";
		lootTag[] = {"Civilian","Work"};
		itemSize[] = {1,2};
		weight = 1;
		varQuantityInit = 1;
		varQuantityMin = 0;
		varQuantityMax = 500;
		varQuantityDestroyOnMin = 1;
		destroyOnEmpty = 1;
		absorbency = 1;
		hiddenSelections[] = {"WholeObject"};
		hiddenSelectionsTextures[] = {"TM\Trader\data\ruble1_co.paa"};
		class DamageSystem
		{
			class GlobalHealth
			{
				class Health
				{
					hitpoints = 100;
					healthLevels[] = {{1.0,{"TM\Trader\data\ruble.rvmat"}},{0.7,{"TM\Trader\data\ruble.rvmat"}},{0.5,{"TM\Trader\data\ruble_damage.rvmat"}},{0.3,{"TM\Trader\data\ruble_damage.rvmat"}},{0.0,{"TM\Trader\data\ruble_destruct.rvmat"}}};
				};
			};
		};
	};
	class MoneyRuble5: MoneyRuble1
	{
		displayName = "$STR_tm_5_ruble_note";
		hiddenSelectionsTextures[] = {"TM\Trader\data\ruble5_co.paa"};
	};
	class MoneyRuble10: MoneyRuble1
	{
		displayName = "$STR_tm_10_ruble_note";
		hiddenSelectionsTextures[] = {"TM\Trader\data\ruble10_co.paa"};
	};
	class MoneyRuble25: MoneyRuble1
	{
		displayName = "$STR_tm_25_ruble_note";
		hiddenSelectionsTextures[] = {"TM\Trader\data\ruble25_co.paa"};
	};
	class MoneyRuble50: MoneyRuble1
	{
		displayName = "$STR_tm_50_ruble_note";
		hiddenSelectionsTextures[] = {"TM\Trader\data\ruble50_co.paa"};
	};
	class MoneyRuble100: MoneyRuble1
	{
		displayName = "$STR_tm_100_ruble_note";
		hiddenSelectionsTextures[] = {"TM\Trader\data\ruble100_co.paa"};
	};
	class HouseNoDestruct;
	class Land_RoadCone: HouseNoDestruct
	{
		scope = 2;
		model = "TM\Trader\Road_Cone.p3d";
	};
	class Hoodie_ColorBase;
	class Hoodie_GraffitiTiles: Hoodie_ColorBase
	{
		scope = 2;
		visibilityModifier = 0.95;
		hiddenSelectionsTextures[] = {"","","","","TM\Trader\data\hoodie_graffiti_co.paa","TM\Trader\data\hoodie_graffiti_co.paa","TM\Trader\data\hoodie_graffiti_co.paa"};
	};
	class Hoodie_DrJ0nes: Hoodie_ColorBase
	{
		scope = 2;
		visibilityModifier = 0.95;
		hiddenSelectionsTextures[] = {"","","","","TM\Trader\data\hoodie_drj0nes_co.paa","TM\Trader\data\hoodie_drj0nes_co.paa","TM\Trader\data\hoodie_drj0nes_co.paa"};
	};
	class VehicleKeyBase: Inventory_Base
	{
		scope = 2;
		displayName = "$STR_tm_vehicle_key";
		descriptionShort = "$STR_tm_vehicle_key_description";
		model = "TM\Trader\keyLada.p3d";
		rotationFlags = 17;
		lootCategory = "Materials";
		lootTag[] = {"Civilian","Work"};
		weight = 4;
		itemSize[] = {1,1};
		fragility = 0.01;
		hiddenSelections[] = {"WholeObject"};
		class DamageSystem
		{
			class GlobalHealth
			{
				class Health
				{
					hitpoints = 100;
					healthLevels[] = {{1.0,{"TM\Trader\data\keyLada.rvmat"}},{0.7,{"TM\Trader\data\keyLada.rvmat"}},{0.5,{"TM\Trader\data\keyLada_damage.rvmat"}},{0.3,{"TM\Trader\data\keyLada_damage.rvmat"}},{0.0,{"TM\Trader\data\keyLada_destruct.rvmat"}}};
				};
			};
		};
	};
	class VehicleKeyDuplicate: VehicleKeyBase
	{
		displayName = "Duplicated Key";
		descriptionShort = "Duplicates the Vehicle Key that you are holding in your Hands.";
	};
	class VehicleKeyLost: VehicleKeyBase
	{
		displayName = "Lost Key";
		descriptionShort = "Creates a Key for a Vehicle in case you lost it. Works also for Vehicles that never had a Key before.";
	};
	class VehicleKeyRed: VehicleKeyBase
	{
		hiddenSelectionsTextures[] = {"TM\Trader\data\keyLadaRed_co.paa"};
	};
	class VehicleKeyBlack: VehicleKeyBase
	{
		hiddenSelectionsTextures[] = {"TM\Trader\data\keyLadaBlack_co.paa"};
	};
	class VehicleKeyGrayCyan: VehicleKeyBase
	{
		hiddenSelectionsTextures[] = {"TM\Trader\data\keyLadaGrayCyan_co.paa"};
	};
	class VehicleKeyYellow: VehicleKeyBase
	{
		hiddenSelectionsTextures[] = {"TM\Trader\data\keyLadaYellow_co.paa"};
	};
	class VehicleKeyPurple: VehicleKeyBase
	{
		hiddenSelectionsTextures[] = {"TM\Trader\data\keyLadaPurple_co.paa"};
	};
};
